osf_tests/test_node.py

```diff
@@ -3783,6 +3784,21 @@ def session(self, user, request_context):
         set_session(s)
         return s
 
+    @pytest.fixture()
+    def collection(self):
+        collection_provider = CollectionProviderFactory()
+        return CollectionFactory(provider=collection_provider)
+
+    @pytest.fixture()
+    def node_in_collection(self, collection):
+        node = ProjectFactory()
```

Comment:
Isn't the default a private node? Should this include an `is_public=True` parameter?